components/app/common/ContactUs/ContactUsForm.js

```diff
@@ -103,7 +103,7 @@ class ContactUsForm extends React.Component {
           </Field>
 
           <div className="actions-container -align-right">
-            <button type="submit" className="c-btn -primary">
+            <button type="submit" className={`c-btn -primary ${this.state.submitted ? '-disabled' : null}`} disabled={this.state.submitted}>
```

Comment:
Should we disable the form after submitting or during the submit? I mean, I think that we should disable the button with the `this.state.submitting`

Reply:
makes sense, will change the logic so its disabled when submitting